configs/projects/pe-installer-runtime.rb

```diff
@@ -95,7 +96,7 @@
     'no-dtls1',
     'no-idea',
     'no-seed',
-    'no-ssl2-method',
+    # 'no-ssl2-method',
```

Comment:
should this be removed entirely?

Reply:
Maybe? I was just copy-pastaing from the pdk-runtime. 🤷‍♂️